src/Symfony/Component/Translation/CatalogueBag/CatalogueBagInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\CatalogueBag;
+
+use Symfony\Component\Translation\MessageCatalogueInterface;
+
+/**
+ * CatalogueBagInterface.
```

Comment:
Could you add a short description what a `CatalogueBag` is? If this is the façade where you can easily get an entire MessageCatalogue chain loaded, wouldn't `MessageCatalogueProvider` or similar be a better name?